### DISCUSSION TITLE
Welcome to llm-app Discussions!

### DISCUSSION THREAD

**Discussion by dxtrous** [pathwaycom/llm-app#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of Pathway's community interested in LLM Applications. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Pipboyguy** [comment]
Hi I'm marcel, a passionate data engineer who dabbles in data science. I intend to use the power of pathway to build something incredible with LLMs
